.env

```diff
@@ -0,0 +1,2 @@
+FACEBOOK_APP_ID = 2480208355596546
+FACEBOOK_APP_SECRET = 027e05a9ac066b8852f3829591d31dd7
```

Comment:
# I have removed the unwanted files. I Will merge the conflict when the project is approved.

Reply:
Hi @Forison, you will need the `,env` file locally, it is not advisable to delete it, I would, however, suggest you add to `gitignore` to prevent it from been checked in. See main reviews for details.